contentcuration/contentcuration/frontend/channelList/views/Channel/ChannelModal.vue

```diff
@@ -2,20 +2,26 @@
 
   <VDialog
     ref="dialog"
-    :value="$route.params.channelId == channelId"
+    :value="channelId && routeParamID === channelId"
     attach="body"
     fullscreen
     scrollable
     transition="dialog-bottom-transition"
   >
     <VCard>
       <VToolbar card prominent dark color="primary">
-        <VBtn icon class="hidden-xs-only" @click="close">
-          <VIcon>clear</VIcon>
+        <VBtn icon @click="close">
+          <VIcon class="notranslate">
+            clear
+          </VIcon>
         </VBtn>
-        <VToolbarTitle>
-          {{ channel.name }}
+        <VToolbarTitle v-if="isNewChannel">
```

Comment:
For nodes, it's a bit more straightforward as you could be adding multiple new nodes at a time. For channels, it will probably be very different as it's only one at a time. Maybe it makes sense to just default to "editing channel"? I agree though, something to hash out once we add autosaving